### DISCUSSION TITLE
Changing cloud effect particle effects

### DISCUSSION THREAD

**Discussion by MrGeoTech** [Minestom/Minestom#629] [Help] [answered]
Is there even a way to change the particle effect of the areacloudentitymeta? Am I looking in the wrong spot? All I need is a few particles on one spot.

**Comment by emortaldev** [comment, marked as answer]
You can change the colour but not the particle afaik

**Comment by RealGatt** [comment]
You should be able to- its part of the entities Meta Data. I can't however find any implementation within Minestom for it.

https://wiki.vg/Entity_metadata#Area_Effect_Cloud

edit; it is in the class- its just been commented out

```
//    public ParticleWrapper getParticle() {
//        return super.metadata.getIndex((byte) 10, new ParticleWrapper(Particle.EFFECT, null));
//    }
//
//    public void setParticle(ParticleWrapper value) {
//        super.metadata.setIndex((byte) 11, Metadata.Particle(value));
//    }
```
